doc/chips/11.rst

```diff
@@ -5,15 +5,17 @@ Status:
   Rejected
 
 Authors:
-  Kyle Brady, Lydia Duncan, Michael Ferguson, Mike Noakes, Vassily Litvinov (for Syntax 2);
-  Vassily Litvinov (for Child Fields and Initialization Completion)
+  Kyle Brady, Lydia Duncan, Michael Ferguson, Mike Noakes, Vassily Litvinov
 
 Abstract
 --------
 
-This document contains the features we discussed as part of the
+This document contains some features we discussed as part of the
 initializer story and chose not to incorporate.
 
+.. contents:: Outline
+   :local:
+
```

Comment:
This does not actually do anything in the github rst version: https://github.com/vasslitvinov/chapel/blob/78346ec00618b367157947a22249aacdd21a9b3f/doc/chips/11.rst